metron-platform/metron-parsers/src/test/java/org/apache/metron/parsers/integration/ParserDriver.java

```diff
@@ -0,0 +1,168 @@
+/**
```

Comment:
Can we get the start of a readme, maybe metron-parsers/integration_tests.md that explains what we are doing here?